recipe/meta.yaml

```diff
@@ -144,11 +145,12 @@ outputs:
         - healpy >=1.16.6
         - healsparse >=1.8.1
         - hpgeom >=1.2.0
-        - htcondor >=23.0,<23.1
+        - htcondor >=23.0
```

Comment:
The maximum here was explicitly to stay on the long-term support branch.  Check with Greg Daues about whether we should change this.

Reply:
Looks like 24.0.x is the LTS version now. @michellegower do you have any opinion on this?

Reply:
@MichelleGower that is the purpose of this initial phase where we release a development version of rubinenv for testing. If you tell us you want to test with 24.0.x we can specify that here and there will be a couple of weeks to test it.

Reply:
I checked with Nick.  Yes, we'd like to test with 24.0.x